### PR TITLE
+choosealicense.com

### PR DESCRIPTION
**Pull request by waldyrious**
ref: https://github.com/github/choosealicense.com/pull/227

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +61,7 @@ The real-life examples that uses the code *isogram* generates in their pages
 | ----------------------------- | ---------------------------- | --------------------------- |
 | apiDoc                        | [apidocjs.com][apidoc]       | `a` `p` `i` `d` `o` `c`     |
 | BrowserSync                   | [browsersync.io][bsync]      | `s` `y` `n` `c` `I` `t`     |
+| Choose a License              | [choosealicense.com][license]| `L` `I` `C` `e` `N` `S` `E` |
```
Comment (shinnn) [inline README.md]: Update this line to:

```
| choosealicense.com            | [choosealicense.com][license] | `L` `I` `C` `e` `N` `S` `E` |
```

Just copy and paste it.

**Comment by shinnn** [comment]
Could you update the commit message to `add choosealicense.com to the showcase`?

File: README.md (hunk 1 above)
Comment (waldyrious) [inline README.md]: wouldn't that misalign the table's separators? Perhaps an extra space could be added to all cells in the second column to compensate.

**Comment by waldyrious** [comment]
I don't have a machine with git installed handy, and github's web interface is still rather limited in terms of what can be done, e.g. amending commits; I'd rather just create a separate PR, it's easier for such a small change. Otherwise, if you don't mind having one less entry in the contributors list, you can make the commit directly, and I'll close this PR.

**Comment by shinnn** [comment]
Landed in https://github.com/shinnn/isogram/commit/00db7f323712ee8de111ba4292a8e288fe653243 (sorry, including my own tweaks).

Thanks, @waldyrious.

**Pull request closed without merging**